local_planner/src/nodes/common.cpp

```diff
@@ -179,31 +156,18 @@ void wrapAngleToPlusMinusPI(double& angle) {
   while (angle < -M_PI) {
     angle += 2 * M_PI;
   }
-  while (angle > M_PI) {
-    angle -= 2 * M_PI;
-  }
-  while (angle < -M_PI) {
-    angle += 2 * M_PI;
-  }
 }
 
 double getAngularVelocity(double desired_yaw, double curr_yaw) {
   wrapAngleToPlusMinusPI(desired_yaw);
   double yaw_vel1 = desired_yaw - curr_yaw;
   double yaw_vel2;
-  if (yaw_vel1 > 0) {
-    yaw_vel2 = -(2 * M_PI - yaw_vel1);
-  } else {
-    yaw_vel2 = 2 * M_PI + yaw_vel1;
-  }
+  //finds the yaw vel for the other yaw direction
+  yaw_vel1>0 ? yaw_vel2 = -(2 * M_PI - yaw_vel1) : yaw_vel2 = 2 * M_PI + yaw_vel1;
 
   double vel;
-  if (std::abs(yaw_vel1) < std::abs(yaw_vel2)) {
-    vel = yaw_vel1;
-  } else {
-    vel = yaw_vel2;
-  }
-
+  // check which yaw direction is shorter
+  std::abs(yaw_vel1) <= std::abs(yaw_vel2)? vel = yaw_vel1: vel = yaw_vel2;
```

Comment:
This should be
` vel = (std::abs(yaw_vel1) <= std::abs(yaw_vel2))? yaw_vel1 : yaw_vel2;`